srcpkgs/cross-x86_64-w64-mingw32/template

```diff
@@ -23,15 +23,15 @@ distfiles="
  http://isl.gforge.inria.fr/isl-${_isl_version}.tar.bz2
  ${SOURCEFORGE_SITE}/project/mingw-w64/mingw-w64/mingw-w64-release/mingw-w64-v${_mingw_version}.tar.bz2"
 checksum="
- 1509dff41369fb70aed23682351b663b56db894034773e6dbf7d5d6071fc55cc
+ de38b15c902eb2725eac6af21183a5f34ea4634cb0bcef19612b50e5ed31072d
  196c3c04ba2613f893283977e6011b2345d1cd1af9abeac58e916b1aab3e0080
  6985c538143c1208dcb1ac42cedad6ff52e267b47e5f970183a3e75125b43c2e
  87b565e89a9a684fe4ebeeddb8399dce2599f9c9049854ca8c0dfbdea0e21912
  67874a60826303ee2fb6affc6dc0ddd3e749e9bfcb4c8655e3953d0458a6e16e
  d59726f34f7852a081fbd3defd1ab2136f174110fc2e0c8d10bb122173fa9ed8
- 5527e1f6496841e2bb72f97a184fc79affdcd37972eaa9ebf7a5fd05c31ff803"
+ 805e11101e26d7897fce7d49cbb140d7bac15f3e085a91e0001e80b2adaf48f0"
 
-only_for_archs="i686 x86_64"
+archs="i686 x86_64"
```

Comment:
Pls no merge. I need to check musl builds and "nostrip=yes" (below). Something isn't right it seems.

Reply:
Please also check i686. It is broken at the moment. There also is no previous version available on the repos for i686 so it probably never worked.

Reply:
Could you elaborate on "broken", it doesn't build, it doesn't work or what?

Reply:
doesn't build

Reply:
I don't see this with 6.0.0 with `./xbps-src ... binary-boostrap i686` and consecutive build with `./xbps-src ... pkg ...` but i noticed that on i686 32bits package is being (needlessly) built. I'll look into this.

I'm generally OK if i686 package will remain disabled because cross-compiler for Win32 is being built for x86_64 host anyway. My only concern is that disabling packages like that hides them from people who's doing `xbps-query` for MinGW.

But i assume it would be best if someone who's actually using this package on appropriate host could build this package and verify if everything is alright.

Reply:
Strip issues for various archs along with redundant 32bits packages should be resolved in 
3fda73a now.